db/data.js

```diff
@@ -1465,9 +1458,9 @@ const produce = [
   {
     foodName: "Venison",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175322/Seasonal/036-deer_nccggr.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "In Britain today, venison specifically refers to deer meat, though in other countries and historically, it could refer to game from any furred animal, or animals like elk and reindeer. It is a dark, lean and tender meat, and age does affect the meat: the younger the animal, the more tender its meat. Additional factors like diet, and how the animal was raised and how it was handled after it was killed, can all affect its meat. Farmed venison is available year round, as is wild roe deer. Wild red and fallow deer are only in season from late October through mid-February, however. Cuts of venison should be chosen according to how you want to cook it, as different cuts from different parts of the animal will have different qualities that will better suit roasting, grilling, barbecuing, or stewing processes, and so on. The market for venison is steadily growing as consumers seek alternatives to traditionally farmed animals like chicken and cows. As deer have only been farmed in the past few decades, and are not domesticated animals, the farming process for many suppliers tends to be less intense and free range. However, increases to scale farming to match demand will not necessarily scale sustainability, as the energy demands of intensive meat farming remain regardless of the meat which is being farmed.",
```

Comment:
Should we remove the comma before "how the animal was raised"?

Reply:
Added more commas, will commit changes!